package.json

```diff
@@ -1,5 +1,17 @@
 {
-  "dependencies": {
-    "@playwright/test": "^1.23.3"
+  "devDependencies": {
+    "@playwright/test": "^1.23.4",
+    "prettier": "~2.7.0",
+    "type-coverage": "~2.22.0",
```

Comment:
Since typescript has optional typing, this checks to ensure every variable/identifier has a type